package.json

```diff
@@ -66,6 +66,10 @@
     "docs": "npm run jsdoc ; npm run otherdoc",
     "prepare": "node ./node_modules/bower/bin/bower install && node ./node_modules/gulp/bin/gulp.js install"
   },
+  "engines": {
```

Comment:
I'm a fan of this too, but perhaps tag it with majors? i.e `8.x.x`

Reply:
Sounds good! 👍